src/plugins/safeSnap/components/HandleOutcome.vue

```diff
@@ -101,21 +106,23 @@ const claimBond = async () => {
     );
     await clamingBond.next();
     actionInProgress.value = null;
-    pendingCount.value++;
+    // updatePendingTransaction(txPendingId, { txId: '' })
```

Comment:
Why is this commented out?

Reply:
This PR is a draft, extract tx ids from safeSnap need to be implemented.
Need your review about the general approach of this PR 🙂

Reply:
Okay thanks! I will have a closer look tomorrow.

Reply:
The general approach looks great